example/ios/App/App/capacitor.config.json

```diff
@@ -8,6 +8,7 @@
 	},
 	"packageClassList": [
 		"AppcuesPlugin",
+		"Appcues",
```

Comment:
is this getting added on your project too?

Reply:
no 🤔

Reply:
It get's removed for me following all the steps here: https://github.com/appcues/appcues-capacitor-plugin/blob/2d83bfc5c7616ea35ec8005dd97f789d34b97274/example/README.md?plain=1#L7-L35 (specifically, the `npx cap sync`)